app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -103,7 +103,12 @@ def self.provider_friendly_name
     private
 
     def client
-      @client ||= ::Fog::Compute.new(:provider => 'google', :google_project => project, :google_client_email => email, :google_key_location => key_path)
+      @client ||= ::Fog::Compute.new(
+        :provider            => 'google',
+        :google_project      => project,
+        :google_client_email => email,
+        :google_key_location => key_path,
+        :instrumentor        => FogExtensions::Debug::DebugHttpInstrumentor)
```

Comment:
This may result in double-logging, since google-api-client already opportunistically uses Rails.logger if available at debug level.  We might want to simply set Google::APIClient.logger in an initialiser.